vispy/visuals/tests/test_volume.py

```diff
@@ -73,14 +73,14 @@ def test_set_data_does_not_change_input():
     for dtype in ['uint8', 'int16', 'uint16', 'float32', 'float64']:
         vol_copy = np.array(vol, dtype=dtype, copy=True)
         # setting clim so that normalization would otherwise change the data
-        V.set_data(vol_copy, clim=(0, 200))
+        V.set_data(vol_copy, clim_range=(0, 200))
```

Comment:
here's an example of the API breakage... though technically, with the `if "clim" in kwargs` conditional that I added to `Volume.set_data` this test would have still worked